info/GameTestList.md

```diff
@@ -36,3 +36,4 @@ These are subject to change at any time, so keep your eye out!
 | Team Fortress 2          | ✔️    | It can move your mouse and will mess with the camera so you may get banned on some servers due to admins, VAC hasn't banned anyone for it yet |
 | Terraria                 | ✔️    |                                                                                                  |
 | World of Warcraft        | ✔️    |                                                                                                |
+|Crossout                  | ✔️    | In windows mode, the goose would drag out the mouse, causing user click outside the windows      |
```

Comment:
Would you mind moving this above CS:GO so that the list is alphabetically sorted?